docs/GettingStarted.md

```diff
@@ -88,7 +89,8 @@ Available settings are:
 * ``debug``: set to true to enable debug logging.
 * ``trace_agent_hostname``: set the hostname of the trace agent.
 * ``trace_agent_port``: set the port the trace agent is listening on.
-* ``env``: set the environment. Defaults to the Rails environment
+* ``use_rails_env``: guess the Datadog environment from the Rails environment.
+* ``env``: set the environment. Overridden by Rails environment if ``use_rails_env`` is set.
```

Comment:
Not totally sure that is good to have two different settings to provide (and override) the same thing. If using the `Rails.env` could create issues, I'll probably go with a default `env: nil` and let the user to be explicit.